src/components/AppInvalidGuest.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+    <Content app-name="cookbook">
+        <!-- <AppNavi class="app-navigation" /> -->
+        <AppContent>
+            <div class="main">
+                <div class="dialog">
+                    <div class="message">
+                        {{ t('cookbook', 'Cannot access recipe folder.') }}
+                    </div>
+                    <div class="explanation">
```

Comment:
class `explanation` is not used and may be removed

```suggestion
                    <div>
```